Sources/SwiftDocC/Semantics/Metadata/PageImage.swift

```diff
@@ -44,4 +44,30 @@ public final class PageImage: Semantic, AutomaticDirectiveConvertible {
     init(originalMarkup: BlockDirective) {
         self.originalMarkup = originalMarkup
     }
+
+    /// Don't use this outside of ``OutOfProcessReferenceResolver/entity(with:)`` .
+    ///
+    /// Directives aren't meant to be created from non-markup but the out-of-process resolver needs to create a ``PageImage`` to associate topic
+    /// images with external pages. This is because DocC renderers external content in the local context. (rdar://78718811)
+    /// https://github.com/apple/swift-docc/issues/468
+    ///
+    /// This is intentionally defined as an underscore prefixed static function instead of an initializer to make it less likely that it's used in other places.
+    static func _make(originalMarkup: BlockDirective, purpose: Purpose, source: ResourceReference, alt: String?) -> PageImage {
+        // FIXME: https://github.com/apple/swift-docc/issues/468
+        return PageImage(
+            originalMarkup:  originalMarkup,
```

Comment:
Since we don't expect the `BlockDirective` to really be used here, maybe we should just fill it in automatically to minimize confusion?

```suggestion
    static func _make(purpose: Purpose, source: ResourceReference, alt: String?) -> PageImage {
        // FIXME: https://github.com/apple/swift-docc/issues/468
        return PageImage(
            originalMarkup:  BlockDirective(name: "PageImage"..),
```

Reply:
I'm fine either way. The call site already needs to do this for the "Metadata" directive so I don't think it matters much if it does so for both directives. The intention is that no other place should need to do this.